policyengine_canada/parameters/gov/provinces/bc/benefits/bcfb/child_eligible_age_threshold.yaml

```
@@ -0,0 +1,9 @@
+description: Child Age eligibility threshold for the British Columbia family benefit.
+values:
+  2022-07-01: 18
+metadata:
+  unit: year
+  label: British Columbia family benefit Child Age eligibility threshold.
```

Please also rename the file to `child_ineligible_age_threshold` since it's non-inclusive
```suggestion
  label: British Columbia family benefit child age ineligibility threshold
```